src/test/java/com/google/gcp_variant_transforms/TestEnv.java

```diff
@@ -0,0 +1,21 @@
+// Copyright 2020 Google LLC
+
+package com.google.gcp_variant_transforms;
+
+import com.google.gcp_variant_transforms.library.VcfParser;
+import com.google.gcp_variant_transforms.library.VcfParserImpl;
+import com.google.gcp_variant_transforms.task.Task;
+import com.google.gcp_variant_transforms.task.VcfToBqTask;
+import com.google.guiceberry.GuiceBerryModule;
+import com.google.inject.AbstractModule;
+
+public class TestEnv extends AbstractModule {
+  
+  @Override
+  protected void configure() {
+    install(new GuiceBerryModule());
+
+    bind(VcfParser.class).to(VcfParserImpl.class);
+    bind(Task.class).to(VcfToBqTask.class);
+  }
+}
```

Comment:
Add new line.

Reply:
This should had been created in Zhiqiang's PR right? Could you sync and update the PR?